frontend/src/tests/components/eMIB/ResponseItem.test.js

```diff
@@ -0,0 +1,207 @@
+import React from "react";
+import { shallow } from "enzyme";
+import ResponseItem, { RESPONSE_TYPE } from "../../../components/eMIB/ResponseItem";
+import LOCALIZE from "../../../text_resources";
+
+it("renders the right response types (reply, reply all, forward)", () => {
```

Comment:
Switch for a describe block

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/172/commits/65fa676ee1553d366b684d85bae2a6d108d7851a